Yoast/wordpress-seo
# Switched react bundle builds to webpack

## Summary

I changed the tooling that bundles react components from `browserify` to `webpack` in order to be able to use advanced webpack loaders in `yoast-components`.

This PR can be summarized in the following changelog entry:

* Switched build tool for bundles that use `yoast-components` to `webpack`

## Relevant technical choices:

* Added two grunt-webpack configurations for production builds and development builds that extend a default configuration
* Removed `configuration-wizard` bundling from `browserify` configuration
* Added `webpack:buildDev` in `build:js` alias and `webpack:buildProd` in `build:release:js` alias after `uglify`, because webpack does it's own minification.

## Test instructions

This PR can be tested by following these steps:

* Run `grunt webpack:buildDev` and `grunt webpack:buildProd`, aswell as `grunt build:js` and `grunt release:js`.
* Additionaly make some temporary changes to the `configuration-wizard` and see if they show up after a javascript build.
